grails-app/controllers/com/unifina/controller/api/RemoveUsersProductsController.groovy

```diff
@@ -0,0 +1,20 @@
+package com.unifina.controller.api
+
+import com.unifina.security.StreamrApi
+import com.unifina.service.ProductService
+import grails.plugin.springsecurity.annotation.Secured
+
+@Secured(["IS_AUTHENTICATED_ANONYMOUSLY"])
+class RemoveUsersProductsController {
+	ProductService productService
+
+	@StreamrApi
+	def index() {
+		if (params.username == null) {
```

Comment:
We need to check that logged-in user `request.apiUser` is an admin. There is a `SecRole` called `ROLE_ADMIN` that indicates that a user is an admin. Take a look at `SecUser#isDevOps` to see how to check for a role.

Reply:
Another option would be to add a new level authentication level to `@StreamrApi`, but I think I will make a separate JIRA task out of that for now.